seqr/views/utils/pedigree_info_utils.py

```diff
@@ -170,10 +182,11 @@ def _convert_fam_file_rows_to_json(column_map, rows, required_columns=None):
         ValueError: if there are unexpected values or row sizes
     """
     required_columns = [JsonConstants.FAMILY_ID_COLUMN, JsonConstants.INDIVIDUAL_ID_COLUMN] + (required_columns or [])
-    missing_cols = set(required_columns) - set(column_map.values())
+    missing_cols = [_to_title_case(_to_snake_case(col)) for col in set(required_columns) - set(column_map.values())]
+    if update_features and JsonConstants.FEATURES not in column_map.values():
+        missing_cols.append('HPO Terms')
```

Comment:
Is 'HPO Terms' the column name expected [here](https://github.com/broadinstitute/seqr/pull/4256/files#diff-a3dbd9acae63c78c7ea2e5514f0945c7068606fe9c3c9688a23c78671ef24ad0R231) in the file header? 

Can we make this a constant?